pages/desktop/frontend/versions.py

```diff
@@ -23,3 +30,94 @@ def versions_page_header(self):
     def latest_version_number(self):
         el = self.find_element(*self._latest_version_locator).text
         return el.split()[1].replace('Version ', '')
+
+    @property
+    def notice_message(self):
+        return self.find_elements(*self._notice_message_locator)[0]
+
+    @property
+    def notice_message_text(self):
```

Comment:
I don't think this is required. The notice `text` could be retrieved from the `notice_message` method.